CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# dbt_recharge version.version
+# dbt_recharge v0.3.1
+[PR #27](https://github.com/fivetran/dbt_recharge/pull/27) includes the following changes:
+
+- For Quickstart users, removed `recharge__line_item_enhanced` from the public models list in `quickstart.yml` since the model is disabled by default.
```

Comment:
Let's add both of these entries under a "Bug Fixes" section.

Reply:
updated